### PR TITLE
Small documentation issues

### PR DESCRIPTION
**Pull request by reebhub**
two documentation issues:
RDoc-1744 - "x509certificate" to "x509certificate2"
RDoc-1746 - added an "Updating Entities" page with a brief explanation about updating entities.

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/server/security/common-errors-and-faq.markdown [hunk 1]
```diff
@@ -306,7 +306,7 @@ The credentials supplied to the package were not recognized (The SSL connection
 2) Another solution is to use the `MachineKeySet` flag during certificate creation:  
 
 {CODE-BLOCK:csharp}
-[DocumentStore].Certificate = new X509Certificate("[path to your pfx file]", 
+[DocumentStore].Certificate = new x509certificate2("[path to your pfx file]",
```
Comment (ppekrol) [inline Documentation/4.2/Raven.Documentation.Pages/server/security/common-errors-and-faq.markdown:309]: should be uppercase

File: Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Session: Updating Entities
+
+To update an entity retrieve it ([explicitly](../../client-api/session/loading-entities#load) 
+or by a [query](../../client-api/session/querying/how-to-query#session.query)), edit the properties 
+you wish to change, and call `saveChanges()` to apply the changes.  
+
+---
+
+{PANEL:Samples}
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown:9]: This is not how other parts of the docs are written. Please review them (by heart) and unify the styling.

File: Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Session: Updating Entities
+
+To update an entity retrieve it ([explicitly](../../client-api/session/loading-entities#load) 
+or by a [query](../../client-api/session/querying/how-to-query#session.query)), edit the properties 
+you wish to change, and call `saveChanges()` to apply the changes.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown:5]: fixed

File: Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Session: Updating Entities
+
+To update an entity retrieve it ([explicitly](../../client-api/session/loading-entities#load)
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown:3]: retrieve it (by [load](../../client-api/session/loading-entities#load) or [query](../../client-api/session/querying/how-to-query#session.query))

File: Documentation/4.2/Raven.Documentation.Pages/server/security/common-errors-and-faq.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/server/security/common-errors-and-faq.markdown:309]: fixed

File: Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/session/updating-entities.markdown]: done

**Pull request merged**